### PR TITLE
Remove redundant experiment_id from ensemble

### PR DESCRIPTION
**Pull request by xjules**
**Issue**
`experiment_id` is not used anymore in `ensemble.evaluate` anymore and thus candidate for removal.

**Approach**
:scissors: 

(Screenshot of new behavior in GUI if applicable)

- [ ] PR title captures the intent of the changes, and is fitting for release notes.
- [ ] Added appropriate release note label
- [ ] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [ ] Make sure tests pass locally (after every commit!)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

File: src/ert/ensemble_evaluator/_ensemble.py [hunk 1]
```diff
@@ -29,12 +29,7 @@
 from ._wait_for_evaluator import wait_for_evaluator
 from .config import EvaluatorServerConfig
 from .identifiers import EVTYPE_ENSEMBLE_FAILED, EVTYPE_ENSEMBLE_STARTED
-from .snapshot import (
-    ForwardModel,
-    RealizationSnapshot,
-    Snapshot,
-    SnapshotDict,
-)
+from .snapshot import ForwardModel, RealizationSnapshot, Snapshot, SnapshotDict
```
Comment (berland) [inline src/ert/ensemble_evaluator/_ensemble.py:32]: unrelated change? (will cause conflict with jonathans pr..)

Comment (xjules) [reply src/ert/ensemble_evaluator/_ensemble.py:32]: will turn it back

**Review by berland — APPROVED** [review]
Probably good! 🙏🏻
